src/plugins/intel_gpu/tests/unit/shape_infer/roi_align_si_test.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "test_utils.h"
+
+#include <intel_gpu/primitives/input_layout.hpp>
+#include <intel_gpu/primitives/bucketize.hpp>
+#include <intel_gpu/primitives/data.hpp>
+
+#include "roi_align_inst.h"
+
+#include "program_wrapper.h"
+
+#include <cmath>
+#include <algorithm>
+
+using namespace cldnn;
+using namespace ::tests;
+
+namespace shape_infer_tests {
+
+struct roi_align_test_params {
+    layout input_layout;
+    int num_roi;
+    size_t pooled_h;
+    size_t pooled_w;
```

Comment:
Please update them as below to fix the Windows build failure:

```suggestion
    int pooled_h;
    int pooled_w;
```

Reply:
Fixed it. thanks.